core-web/libs/portlets/dot-content-drive/ui/tsconfig.json

```diff
@@ -1,13 +1,11 @@
 {
     "compilerOptions": {
         "target": "es2022",
+        "useDefineForClassFields": false,
         "forceConsistentCasingInFileNames": true,
-        "strict": true,
```

Comment:
😂

Reply:
Yeah, all the test of this project were blocked just because of errors on other internal libs